content/blog/posts/2022-05-19-five-tips-to-improve-your-terminal-experience.md

```diff
@@ -0,0 +1,70 @@
+---
+authorId: rwwagner90
+categories:
+  - devtools
+date: '2022-05-19'
+slug: five-tips-to-improve-your-terminal-experience
+title: 5 Tips to Improve Your Terminal Experience
+---
+
+We've come a long way from the stereotypical terminals seen in the movies with
+green text on black backgrounds and developers furiously
+slinging commands from memory - all on an eight inch screen.
+
+There are now lots of great options for improving both the style and functionality
```

Comment:
While our day-to-day development process might not appear as exciting and action-packed as the silver screen renditions, we software developers do have a lot of great options to streamline our processes by improving the stye and functionality of our terminals.